src/OrchardCore.Modules/OrchardCore.XmlRpc/Controllers/HomeController.cs

```diff
@@ -54,7 +54,7 @@ public async Task<IActionResult> ServiceEndpoint([ModelBinder(BinderType = typeo
             result.Save(w);
         }
 
-        var content = Encoding.UTF8.GetString(stream.GetBuffer(), 0, (int)stream.Length);
+        var content = Encoding.UTF8.GetString(stream.GetReadOnlySequence());
```

Comment:
Didn't know about these, apparently there are a few in `EncodingExtensions`. We should check if there aren't other places where we could use these.